### ISSUE TITLE
set --write is not taken into account

### ISSUE THREAD

**Issue by cdarken** [sibradzic/upp#2]
Hi.
I'm trying to use your tool to modify some settings in the PP tables for a Polaris card.
But when I try to set them I get this error:
```
WARNING: Nothing was written to '/sys/class/drm/card1/device/pp_table'. Add --write option to commit the changes for real!
```
My command line:
```
sudo ./upp.py -i /sys/class/drm/card1/device/pp_table set --write /MemClockDependencyTable/Entries/2/Vddci=800
```

**Comment by cdarken** [comment]
I think I found the issue. [Here](https://github.com/sibradzic/upp/blob/master/upp.py#L122) you use `'False'` to set the default value and I modified it to `False` as I think it needs to be a boolean.

**Comment by sibradzic** [comment]
Thanks for report @cdarken, yeah, that click option default was wrongly initialized, fixed it in 7272b0b
